plugins/source/azure/client/services/authorization.go

```diff
@@ -1,4 +1,4 @@
-//go:generate mockgen -destination=./mocks/authorization.go -package=mocks . RoleAssignmentsClient,RoleDefinitionsClient
+//go:generate mockgen -destination=./mocks/authorization.go -package=mocks . AuthorizationRoleAssignmentsClient,AuthorizationRoleDefinitionsClient
```

Comment:
Since all mocks are in the same package `mocks` we prefix clients with the service name to avoid collisions